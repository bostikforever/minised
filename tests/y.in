The Captial Letters Should All Get Converted To Lower-Case.
