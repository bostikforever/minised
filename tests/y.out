the captial letters should all get converted to lower-case.
